core/help/help.py

```diff
@@ -657,17 +657,7 @@ def gui_list_help(gui: imgui.GUI):
     total_page_count = len(pages_list)
     # print(pages_list[current_page])
 
-    if total_page_count == 0:
-        page_info = "empty"
-    else:
-        page_info = f"{current_list_page}/{total_page_count}"
-
-    gui.text(f"List: {selected_list} ({page_info})")
-
-    # Extract description from list declaration, i.e. mod.list(..., desc=...))
-    if (desc := registry.decls.lists[selected_list].desc) is not None:
```

Comment:
Your pull request seems to have removed the help code that gets the list description from the registry and displays it. Can you put that back?

Reply:
Please put this back.